src/main/webapp/createYourOwn.html

```diff
@@ -22,6 +22,51 @@
             </nav>
         </div>
       </header>
+      <h1>New Initiative</h1>
+      <hr style="height:4px;background-color:#F8944C;border-width:0px;color:#F8944C;width:300px;margin-bottom:30px">
+      <div id="new-initiative">
+          <form id="newInitiativeForm" action="myUploadUrl" method="POST" enctype="multipart/form-data">
+            <p class="initiative-input">Title:
```

Comment:
Just a note, but you could also use the <label> tag to serve the same purpose here.

Reply:
I don't think I quite understand what you mean.